src/shared/components/palette/email/write-email-field.component.ts

```diff
@@ -13,4 +13,8 @@ export class WriteEmailFieldComponent extends AbstractFieldWriteComponent implem
   ngOnInit() {
     this.emailControl = this.registerControl(new FormControl(this.caseField.value)) as FormControl;
   }
+
+  public inputBlur() {
+    this.emailControl.markAsUntouched();
```

Comment:
Are you sure this is what you want to do? If you enter an invalid value and then leave the field, the errors won't show, based on the logic within the HTML file (`*ngIf="emailControl.errors && emailControl.touched"`).

This bit of code ensures that `emailControl.touched` will always evaluate to `false` when the control doesn't have focus.

Reply:
As discussed changed it to on dirty state